package-lock.json

```diff
@@ -1161,6 +1161,13 @@
       "integrity": "sha1-OciRjO/1eZ+D+UkqhI9iWt0Mdm8=",
       "requires": {
         "hoek": "2.x.x"
+      },
+      "dependencies": {
+        "hoek": {
+          "version": "2.16.3",
```

Comment:
This version is vulnerable, and why is it being added here?

Reply:
See above reply.